comptests/prov/run-prov-sanity-multinode-k8s.groovy

```diff
@@ -13,7 +13,7 @@ pipeline {
 		stage('CODE_CHECKOUT') {
 			steps{
 			    cleanWs()
-			    checkout([$class: 'GitSCM', branches: [[name: '*/main']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'rel_sanity_github_auto', url: 'https://github.com/Seagate/cortx-test.git']]])
+			    checkout([$class: 'GitSCM', branches: [[name: '*/CORTX-NAMESPACE']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'rel_sanity_github_auto', url: 'https://github.com/Seagate/cortx-test.git']]])
```

Comment:
so u will be rebasing this branch every now and then with latest main in case some chnages are done and those are needed?

Reply:
yes once u are going to merge will change it to main as already this small change needed for now.

Reply:
better change before merge so u don't need to raise another PR just for this small change. @seagate-sarang-sawant